### PR TITLE
Warning Fixes: Fixed style issues

### PR DESCRIPTION
**Pull request by veena14cs**
## Explanation
This PR optimizes the code by fixing code styling issues.

## Checklist

- [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by rt4914** [comment]
@veena14cs This PR review will actually be delayed from my side for couple of hours as I am in middle of something. Also, I believe that PR is not blocking anyone so that won't be an issue.

**Review by nikitamarysolomanpvt — APPROVED** [review]
LGTM

File: domain/src/main/java/org/oppia/domain/question/QuestionAssessmentProgressController.kt [hunk 1]
```diff
@@ -93,7 +93,7 @@ class QuestionAssessmentProgressController @Inject constructor(private val dataP
    *     recommended that calling code only listen to this result for failures, and instead rely on [getCurrentQuestion]
    *     for observing a successful transition to another state.
    */
-  fun moveToPreviousQuestion(): LiveData<AsyncResult<Any?>> {
+  private fun moveToPreviousQuestion(): LiveData<AsyncResult<Any?>> {
     check(playing) { "Cannot move to the previous question unless an active training session is ongoing" }
     return MutableLiveData(AsyncResult.success<Any?>(null))
   }
```
Comment (rt4914) [inline domain/src/main/java/org/oppia/domain/question/QuestionAssessmentProgressController.kt]: Do not make changes to this files, as these functions are not getting used yet, but will get used soon.

**Review by rt4914 — APPROVED** [review]
LGTM, just one change suggested.

File: domain/src/main/java/org/oppia/domain/question/QuestionAssessmentProgressController.kt (hunk 1 above)
Comment (veena14cs) [reply domain/src/main/java/org/oppia/domain/question/QuestionAssessmentProgressController.kt]: Done.

**Pull request merged**